marimo/_ast/compiler.py

```diff
@@ -163,6 +167,20 @@ def compile_cell(
         # since there is an actual file to read from.
         cache(filename, code)
 
+    # pytest assertion rewriting, gives more context for assertion failures.
+    if test_rewrite:
+        # pytest is not required, so fail gracefully if needed
+        try:
+            from _pytest.assertion.rewrite import (  # type: ignore
```

Comment:
This is a private API. If no public API can be used, let's handle a much more generic exception than `ImportError` (just `Exception`)